modules/configuring-huge-pages.adoc

```diff
@@ -10,43 +14,66 @@ Node Tuning Operator to allocate huge pages on a specific node.
 tuned profile, which describes how many huge pages should be allocated:
 +
 ----
-$ oc label node <node-using-hugepages> hugepages=true
+$ oc label node <node_using_hugepages> hugepages=true
 ----
 
-. Create a tuned profile for this specific node and allocate some huge pages. To
-associate the profile to the correct node, edit the tuned resource and add a
-`profile` and `match` section:
+. Copy the default tuned profile and customize it to create a tuned profile
+allocating huge pages for your labeled node.
+
+.. Export the default tuned profile and save it to a file (`tuned.yaml`):
 +
 ----
-$ oc edit tuned
+$ oc get tuned --export -o yaml > tuned.yaml
 ----
 
-.. In the `profile` section, add the following tuned profile for allocating huge
-pages:
+.. To rename the profile to `hugepages`, edit the `tuned.yaml` file and update
+the `metadata.name` parameter:
 +
 ----
-- data: |
-    [main]
-    summary=Configuration for hugepages
-    include=openshift-node
-    [vm]
-    transparent_hugepages=never
-    [sysctl]
-    vm.nr_hugepages=1024
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: hugepages <1>
+  namespace: openshift-cluster-node-tuning-operator
 ----
+<1> Rename the profile.
 
-.. To apply the profile to the correct node, add the following to the `match`
+.. To add a tuned profile for allocating huge pages and associate the profile to
+the correct node, add the following `profile` and `match` sections to the `spec`
 section:
 +
 ----
- - match:
-   - label: hugepages
-   priority: 30
-   profile: hugepages
+spec:
+  profile: <1>
+  - data: |
+      [main]
+      summary=Configuration for hugepages
+      include=openshift-node
+
+      [vm]
+      transparent_hugepages=never
+
+      [sysctl]
+      vm.nr_hugepages=1024
+    name: node-hugepages
+  recommend:
+  - match: <2>
+    - label: hugepages
+    priority: 30
+    profile: node-hugepages
+----
+<1> Add a `profile` section.
+<2> Add a `match` section.
+
+. Create the new, custom  `hugepages` tuned profile using the modified
+`tuned.yaml` file:
++
+----
+$ oc create -f tuned.yaml
 ----
 
-The Operator applies the new profile to the correct node and allocates huge
-pages:
+After completing this procedure, the Operator applies the new profile to the
```

Comment:
Is this the result of running the previous step? Is this output displayed after you run the `oc create` command? If so, I'd connect the remainder of the module to the previous step and ~After completing this procedure,~.

Reply:
No, I believe these are other logs. Would be good to show the command being run to view these logs here.

Reply:
For that, you need to know which containerized tuned pod to query. You can do something like `oc logs <tuned_pod_on_node_using_hugepages> -n  openshift-cluster-node-tuning-operator | tail -n2` and with 4.2 you'll get
```
2019-08-08 07:20:41,276 INFO     tuned.plugins.plugin_sysctl: reapplying system sysctl
2019-08-08 07:20:41,286 INFO     tuned.daemon.daemon: static tuning from profile 'node-hugepages' applied
```

Note that in 4.2, containerized tuned no longer runs in one-shot mode, so you'll not get the ` tuned.daemon.daemon: terminating Tuned` line, so it should be dropped.  Personally, I'd say something like this in the docs:

```
$ oc logs <tuned_pod_on_node_using_hugepages> -n  openshift-cluster-node-tuning-operator | grep 'applied$' | tail -n1
2019-08-08 07:20:41,286 INFO     tuned.daemon.daemon: static tuning from profile 'node-hugepages' applied
```

Then the log output will be valid for 4.1 and 4.2